sunkit_image/radial.py

```diff
@@ -430,10 +432,14 @@ def fnrgf(
         A two-dimensional array of bin edges of size ``[2, nbins]`` where ``nbins`` is the number of bins.
     order : `int`
         Order (number) of fourier coefficients and it can not be lower than 1.
-    attenuation_coefficients : `float`
-        A two dimensional array of shape ``[2, order + 1]``. The first row contain attenuation
-        coefficients for mean calculations. The second row contains attenuation coefficients
-        for standard deviation calculation.
+    range_mean : `list`, optional
```

Comment:
I think if we plan to add them as inputs, we need to work on the name.

Reply:
Yes that would be great....

this modification may work....

`def _set_attenuation_coefficients(order, mean_attenuation_range=None, std_attenuation_range=None, cutoff=0):



    """
    This is a helper function to Fourier Normalizing Radial Gradient Filter
    (`sunkit_image.radial.fnrgf`).

    This function sets the attenuation coefficients in the one of the following two manners:

    If ``cutoff`` is ``0``, then it will set the attenuation coefficients as linearly decreasing between
    the range ``mean_attenuation_range`` for the attenuation coefficients for mean approximation and ``std_attenuation_range`` for
    the attenuation coefficients for standard deviation approximation.

    If ``cutoff`` is not ``0``, then it will set the last ``cutoff`` number of coefficients equal to zero
    while all the others the will be set as linearly decreasing as described above.

    .. note::

        This function only describes some of the ways in which attenuation coefficients can be calculated.
        The optimal coefficients depends on the size and quality of image. There is no generalized formula
        for choosing them and its up to the user to choose a optimum value.

    .. note::

        The returned maps have their ``plot_settings`` changed to remove the extra normalization step.

    Parameters
    ----------
    order : `int`
        The order of the Fourier approximation.
    mean_attenuation_range : `tuple`, optional
        A tuple of length ``2`` which contains the highest and lowest values between which the coefficients for
        mean approximation be calculated in a linearly decreasing manner.
    std_attenuation_range : `tuple`, optional
        A tuple of length ``2`` which contains the highest and lowest values between which the coefficients for
        standard deviation approximation be calculated in a linearly decreasing manner.
    cutoff : `int`, optional
        The numbers of coefficients from the last that should be set to ``zero``.

    Returns
    -------
    `numpy.ndarray`
        A numpy array of shape ``[2, order + 1]`` containing the attenuation coefficients for the Fourier
        coffiecients. The first row describes the attenustion coefficients for the Fourier coefficients of
        the mean approximation. The second row contains the attenuation coefficients for the Fourier coefficients
        of the standard deviation approximation.
    """
    if std_attenuation_range is None:
        std_attenuation_range = (1.0, 0.0)
    if mean_attenuation_range is None:
        mean_attenuation_range = (1.0, 0.0)
    attenuation_coefficients = np.zeros((2, order + 1))
    attenuation_coefficients[0, :] = np.linspace(mean_attenuation_range[0], mean_attenuation_range[1], order + 1)
    attenuation_coefficients[1, :] = np.linspace(std_attenuation_range[0], std_attenuation_range[1], order + 1)

    #A two dimensional array of shape ``[2, order + 1]``. The first row contains attenuation
    #coefficients for mean calculations. The second row contains attenuation coefficients
    #for standard deviation calculation.

    if cutoff > (order + 1):
        msg = "Cutoff cannot be greater than order + 1."
        raise ValueError(msg)

    if cutoff != 0:
        attenuation_coefficients[:, (-1 * cutoff) :] = 0

    return attenuation_coefficients`

Reply:
Sounds good to me.